doc/model.rst

```diff
@@ -0,0 +1,2271 @@
+            instance to itself, we instruct SmartSim to recognize the name of `model_1` as a valid data
+            source for subsequent use in ``Client.set_data_source()``.
+
+        In the application snippet below, we demonstrate running the ML model:
+
+        .. code-block:: python
+
+            # Set the Model source name
+            client.set_data_source("model_1")
+            # Run the ML model
+            client.run_model(name="mnist_cnn", inputs=["mnist_images"], outputs=["Identity"])
+
+        The ``Orchestrator`` now contains prefixed output tensors:
+
+        .. code-block:: bash
+
+            1) "model_1.Identity"
+            2) "model_1.mnist_cnn"
+            3) "model_1.mnist_images"
+
+        .. note::
+            The output tensors are prefixed because we executed ``model_1.enable_key_prefixing()``
+            in the driver script which enables prefixing for tensors, ``Datasets`` and lists.
+
+        **Access ML Models from outside the script they were loaded in**
+
+        SmartSim supports executing prefixed ML models with prefixed tensors sent to the ``Orchestrator`` by separate
+        ``Models``. To achieve this, users need to provide the ``Model`` `name` that stored the ML model and tensor
+        to ``Client.set_data_source()``. This action instructs the ``Client`` to prepend the ``Model``
+        `name` to all key searches. For SmartSim to recognize the ``Model`` `name` as a data source,
+        users must execute the ``Model.register_incoming_entity()`` function on the ``Model``
+        responsible for the search and pass the ``Model`` instance that stored the data.
+
+        In the example, a ``Model`` named `model_1` has placed a ML model and tensor in a standalone
+        ``Orchestrator`` with prefixing enabled on the ``Model``. The contents of the ``Orchestrator``
+        are as follows:
+
+        .. code-block:: bash
+
+            1) "model_1.mnist_cnn"
+            2) "model_1.mnist_images"
+
+        We create a separate ``Model``, named `model_2`, with the executable application code below.
+
+        .. note::
+            In the driver script, after initializing the ``Model`` instance named `model_2`,
+            we execute ``model_2.register_incoming_entity(model_1)``. By passing the producer ``Model``
+            instance to the consumer ``Model``, we instruct SmartSim to recognize the name of `model_1` as a valid data
+            source for subsequent use in ``Client.set_data_source()``.
+
+        In the application snippet below, we demonstrate running the ML model:
+
+        .. code-block:: python
+
+            # Set the Model source name
+            client.set_data_source("model_1")
+            # Run the ML model
+            client.run_model(name="mnist_cnn", inputs=["mnist_images"], outputs=["Identity"])
+
+        The ``Orchestrator`` now contains prefixed output tensors:
+
+        .. code-block:: bash
+
+            1) "model_name.Identity"
```

Comment:
is this model_name or model_1 or model_2?  Same quesiton for the two lines below

Reply:
Same question for the script version of this tab.